configs.py

```diff
@@ -88,7 +89,7 @@
 # --------------------------------
 
 # Specify either a train/ test split, e.g. 0.2 -> 80/ 20 split
-TEST_SPLIT_SIZE = 0.2
+VAL_SPLIT_SIZE = -1
```

Comment:
what does -1 mean?

Reply:
You set it to -1, if you want to use the validation sets instead.

Reply:
maybe complement the comment then?